packages/data/addon/adapters/fact-interface.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
+import NaviFactsModel from 'navi-data/models/navi-facts';
```

Comment:
Ran into prettier issues when trying to import just the type. We might need to upgrade to prettier 2.x